network.go

```diff
@@ -208,7 +216,7 @@ type network struct {
 	attachable   bool
 	inDelete     bool
 	ingress      bool
-	driverTables []string
+	driverTables []networkDBTable
```

Comment:
I have to dig into the changes a bit more. But can you pls confirm if this PR will have any impact with upgrades or interoperability with existing cluster with older versions nodes